### PR TITLE
Components examples

### PR DESCRIPTION
**Pull request by olgamie**
Short description (with a reference to an issue).
#280 this PR updates demo app and fixes `rangeInput` 

**DoD**

- [ ] Major project work has a corresponding task. If there’s no task for what you are doing, create it. Each task needs to be well defined and described.

- [ ] Change has been tested (manually or with automated tests), everything runs correctly and works as expected. No existing functionality is broken.

- [ ] No new error or warning messages are introduced.

- [ ] All interaction with a semantic functions, examples and docs are written from the perspective of the person using or receiving it. They are understandable and helpful to this person.

- [ ] If the change affects code or repo sctructure, README, documentation and code comments should be updated. 

- [ ] All code has been peer-reviewed before merging into any main branch.

- [ ] All changes have been merged into the main branch we use for development (develop).

- [ ] Continuous integration checks (linter, unit tests) are configured and passed.

- [ ] Unit tests added for all new or changed logic.

- [ ] All task requirements satisfied. The reviewer is responsible to verify each aspect of the task.

- [ ] Any added or touched code follows our style-guide.

### CODE REVIEW

File: examples/app.R [hunk 1]
```diff
@@ -142,18 +162,27 @@ accordion <- function() {
   )
 }
 grid <- function() {
+  myGrid <- grid_template(default = list(
```
Comment (krystian8207) [inline examples/app.R:165]: Shouldn't myGrid definitoin be visible in the example?

File: examples/app.R [hunk 2]
```diff
@@ -215,6 +244,20 @@ card_demo <- function() {
     )
   )
 }
+
+msgbox_demo <- function() {
+  div(
+    h1(class="ui header", id="messagebox", "Messagebox"),
+    demo(messagebox(header = "Main header", content = "text")),
```
Comment (krystian8207) [inline examples/app.R:251]: Just the question for future: Is `messagebox` proper name according to our naming guide? Shouldn't that be `message_box`?

File: R/slider.R [hunk 3]
```diff
@@ -98,12 +98,12 @@ range_input <- function(input_id, value, value2, min, max, step = 1, class = NUL
 #' @param inputId Input name.
 #' @rdname slider
 #' @export
-rangeInput <- function(inputId, label, min, max, value, step = 1, width = NULL, ...) {
+rangeInput <- function(inputId, label, min, max, value, value2, step = 1, width = NULL, ...) {
```
Comment (krystian8207) [inline R/slider.R:101]: I've just realized such function doesn;t exist on shiny. I'll create a separate issue regarding that,

**Review by krystian8207 — CHANGES_REQUESTED** [review]
@olgamie lgtm, but please look at the comments.

File: examples/app.R (hunk 1 above)
Comment (olgamie) [reply examples/app.R]: thanks fixed

File: examples/app.R (hunk 2 above)
Comment (olgamie) [reply examples/app.R:250]: I will add issue for this

File: R/slider.R (hunk 3 above)
Comment (olgamie) [reply R/slider.R:101]: yes, we should probably remove this function

**Review by krystian8207 — APPROVED** [review]
lgtm

**Pull request merged**